aldryn_newsblog/managers.py

```diff
@@ -33,6 +38,45 @@ def get_months(self, namespace):
             # Use day=3 to make sure timezone won't affect this hacks'
             # month value. There are UTC+14 and UTC-12 timezones.
             {'date': datetime.date(year=year, month=month, day=3),
-             'count': date_counter[(year, month)]}
+             'num_entries': date_counter[(year, month)]}
             for year, month in dates]
         return months
+
+    def get_authors(self, namespace):
+        """
+        Get authors with articles count for given namespace string.
+
+        Returns Person queryset annotated with and ordered by 'num_entries'.
+        """
+
+        # This methods relies on the facts that:
+        # - each Article has author assigned on save()
```

Comment:
Hmm, in fact we don't care if `author` is required in this place, as we are looking for all articles of a `Person`, not the other way around. So, I'll remove that comment.